src/main/java/teammates/logic/core/FeedbackQuestionsLogic.java

```diff
@@ -533,31 +538,24 @@ public void updateFeedbackQuestionNumber(FeedbackQuestionAttributes newQuestion)
      */
     private void adjustQuestionNumbers(int oldQuestionNumber,
             int newQuestionNumber, List<FeedbackQuestionAttributes> questions) {
-        
-        if (oldQuestionNumber > newQuestionNumber && oldQuestionNumber >= 1) {
-            for (int i = oldQuestionNumber - 1; i >= newQuestionNumber; i--) {
-                FeedbackQuestionAttributes question = questions.get(i - 1);
-                question.questionNumber += 1;
-                try {
+        try {
```

Comment:
Looking at the code, it might be better to use a separate method which contains just the try and catch blocks in the original code, and have both cases use that method. This keeps the amount of code in the try block to a minimum (thereby avoiding catching exceptions from other parts of the code unintentionally), while avoiding the need to duplicate code.